src/test_snmp.cpp

```diff
@@ -86,6 +88,190 @@ TEST_F(SNMPTest, LatencyCalculations)
   delete tbl;
 }
 
+TEST_F(SNMPTest, LatencyScopeCalculations)
+{
+  cwtest_completely_control_time(true);
+
+  // Create a table.  Use a different OID base because otherwise the indices
+  // clash with later tests.
+  SNMP::EventAccumulatorByScopeTable* tbl = SNMP::EventAccumulatorByScopeTable::create("latency", test_oid);
+
+  // Just put one sample in (which should have a variance of 0).  This should
+  // show up in only the current 5 minute stats.
+  tbl->accumulate(100);
+
+  // Check the counts
+  ASSERT_EQ(0, snmp_get(".1.2.2.1.7.1.4.110.111.100.101"));
```

Comment:
Can you add comments for the other two OIDs here as well? and those below. Seems really useful to give them human readable values